src/pages/MenuLayout.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {Box, makeStyles} from '@material-ui/core';
+import bgImage from './mc-bg.png';
+import GameTitle from '../components/ui/GameTitle';
+import IconMenu from '../components/ui/IconMenu';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundImage: `url(${bgImage})`,
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    boxSizing: 'border-box',
+    padding: theme.spacing(4),

```

Comment:
Should we move this to `div#screen`? Is there a screen display we don't want padding on?